smoke-test/urls.txt

```diff
@@ -0,0 +1,10 @@
+# First token represents the path to access, remaining tokens are a
+# comma separated list of text assertions (text we expect to find in the
+# rendered page):
+/ncov, novel coronavirus, Phylogeny, Diversity, Filter by Location
```

Comment:
Because of the single-page-app nature of auspice, I was thinking it's worth asserting against a couple rarely changing strings in the content, otherwise I was finding that we get `200` responses, even though the page might not have actually rendered.

My thinking is, if we pick the right strings, like `coronavirus`, it's unlikely that they change frequently, if they do change, we'll catch it in CI/CD, and just need to update this text file.

Reply:
Ensuring more than just `200` is good (and necessary). These strings on their own are fantastic, but I found myself wondering about the possibilities here, for instance:
If we define the keyword "rectangularTree" here can we check that the DOM has a SVG element with id "MainTree" and groups with ids A, B, C etc. I'm guessing this is going beyond the realm of smoke-testing?!?

Again, just checking for strings as you've done here is a _massive_ step forward for us, and I'm happy to implement this solution but wanted your thoughts on the above and how it fits into testing more generally.

Reply:
We could definitely build out the DSL a bit, make it so there are a few other rules.

A lot of this machinery was already in place in the integration tests, but my thinking (in agreement with yours) is it can be nice to just toss a new line in a text file for this type of assertion, vs., writing an integration tests for each. Also, I think the snapshot based approach, which looks at the whole text content, is better than plucking specific elements from the page (like in the current smoke tests).... **all this is to say**...

We could build out the integration tests further, so that you can programmatically do more complex assertions against the rendered page (and we should have this in mind, if our smoke tests get too complicated). But adding a few more simple rules that someone could place in the text file might be neat.

Reply:
Agreed 👍